.travis.yml

```diff
@@ -35,6 +36,7 @@ before_install:
         conda install pytest pytest-cov codecov -c conda-forge
         pip install pytest-shutil
         pip install git+https://github.com/theochem/iodata.git
+        pip install git+https://github.com/rpmuller/pyquante2.git
```

Comment:
I think what we should do instead of having this added in two separate ways (`pip` installing `master` and `conda` installing an older tagged version) is to have this `pip install git...` line after lines 13 and 14 in `requirements.txt`: https://github.com/cclib/cclib/blob/0217c94cb06e1c812459fc87ee65b877c7ba3599/requirements.txt#L14
so that we can just drop PyQuante (1) later if possible.

Reply:
I've made the changes that you suggested -- PyQuante 1.6.5 should be installed for Python 2.x.x machine in Travis (from sourceforge) and pyquante2 should be installed for Python 3.x.x machine (from github) in Travis.